src/sidebar/components/annotation-omega.js

```diff
@@ -79,14 +62,7 @@ function AnnotationOmega({
     createDraft(annotation, { ...draft, text });
   };
 
-  const onReply = () => {
```

Comment:
This change is unrelated to other PR changes; I can strip it out. This error handling is no longer necessary because `AnnotationActionBar` does the error handling itself.